alacritty/src/config/monitor.rs

```diff
@@ -21,6 +21,10 @@ pub fn watch(mut paths: Vec<PathBuf>, event_proxy: EventLoopProxy<Event>) {
         return;
     }
 
+    // Exclude char devices like `/dev/null`, sockets, and so on, by checking that file type is a
+    // regular file.
+    paths.retain(|path| path.metadata().map_or(false, |metadata| metadata.file_type().is_file()));
```

Comment:
> Tests whether this file type represents a regular file. The result is mutually exclusive to the results of is_dir and is_symlink; only zero or one of these tests may pass.

This is from the Rust docs on `is_file`. This implies that this code would filter symlinks, which is incorrect. So the code should probably be `is_file() || is_symlink()`?

Reply:
No, the `metadate()` call resolves symlinks.

Reply:
Oh word, good catch.